docs/sidebars.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+module.exports = {
+  'docs': {
+    'Introduction': [
+      'introduction/overview',
+      'introduction/community'
+    ],
+    'Architecture': [
+      'architecture/overview'
+    ],
+    'Components': [
```

Comment:
I think we should rename 'Components' to 'Packages' like it is in the architecture doc. Each package will contain one or more components.

Reply:
Great thanks Ryan. Think this was some confusion from my side

Reply:
fixed in https://github.com/openhie/instant/pull/9/commits/a4d9f6fc9d902d1c89579cdeacd8d70ed79e1347